docs/services/gcp-services-guide.md

```diff
@@ -280,7 +280,7 @@ our `mysql-standard` instance the default `MySQLInstanceClass` for namespace
 ```bash
 kubectl -n app-project1-dev label mysqlinstanceclass mysql-standard default="true"
 ```
-For more details about resource classes in Crossplane, see 
+For more details about resource classes in Crossplane, see
```

Comment:
Nitpick: I would normally use `see the` instead of `see`, because it reads better as a complete sentence